app/assets/stylesheets/components/_navbar.scss

```diff
@@ -4,19 +4,23 @@ nav {
   justify-content: space-evenly;
   // position: relative;
   width: 95%;
-  margin: 0 auto;
-  margin-top: 10px;
+  margin: 10px auto 0;
   height: 70px;
   padding: 16px;
   background-color: rgb(232, 230, 230);
   border-radius: 15px;
+  .logo{
+    height: 40px;
+    padding-left: 10px;
 
+  }
```

Comment:
Don't forget to include some spacing between border-radius and .logo and remove the empty line inside .logo